Code Examples/Oleksandr Bazurin/Program.cs

```diff
@@ -0,0 +1,313 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SeleniumBasics2
+{
+    public static class Examples
+    {
+        public static void Main(string[] args)
+        {
+            // Wrapping driver instance in using operator
+            using (IWebDriver driver = new FirefoxDriver())
+            {
+                //XPathExpessions(driver);
+
+                //CssExpessions(driver);
+
+                //ImplicitWaits(driver);
+
+                //ExplicitWaits(driver);
+
+                //SelectElements(driver);
+
+                //Form(driver);
+
+                //Checkbox(driver);
+
+                Practise(driver);
+            }
+        }
+
+        public static void XPathExpessions(IWebDriver driver)
+        {
+            // Navigate to test Url
+            driver.Navigate().GoToUrl($"https://atata-framework.github.io/atata-sample-app/#!/plans");
```

Comment:
It's probably needed to use @ instead of $ when using simple URLs with no variables

Reply:
Brilliant comment.

Reply:
@IhorStechyshyn You are right, mate 💯

Reply:
a comment for task 2